### ISSUE TITLE
Use same message in multiple places

### ISSUE THREAD

**Issue by RodentOfUnusualSize** [RodentOfUnusualSize/random-message-loader#2]
**Is your feature request related to a problem? Please describe.**
It would be cool if a randomly-generated message could be used in multiple places.

For example, if I had a web page with four elements, and I loaded messages from a single URL, I might want different randomly-selected messages from that source in two of the elements, but the other two should always share the same randomly-selected message.

**Describe the solution you'd like**
The best solution is probably an additional attribute, say `data-saria-random-message-id`.

Then you could do:

```html
<!--
  These next two paragraphs will have randomly-selected messages from `messages.txt`.
  They *might* have the same message, or might not. It’s random!
-->
<p data-saria-random-message-src="/messages.txt"></p>
<p data-saria-random-message-src="/messages.txt"></p>

<!--
  These next two paragraphs will have randomly-selected messages from `messages.txt`.
  They will *always* have the same message.
  It might the same message as one or both of the previous paragraphs, or might not. It’s random!
-->
<p data-saria-random-message-src="/messages.txt" data-saria-random-message-id="1"></p>
<p data-saria-random-message-src="/messages.txt" data-saria-random-message-id="1"></p>

<!--
  These next two paragraphs will have randomly-selected messages from `messages.txt`.
  They will *always* have the same message.
  It might the same message as any of the previous paragraphs, or might not. It’s random!
-->
<p data-saria-random-message-src="/messages.txt" data-saria-random-message-id="2"></p>
<p data-saria-random-message-src="/messages.txt" data-saria-random-message-id="2"></p>
```

Basically, whenever multiple elements have the same message source and ID, they will always have the same generated content.

**Describe alternatives you've considered**
None.

**Additional context**
None.

**Comment by RodentOfUnusualSize** [comment]
Being addressed in the [message-ids](/RodentOfUnusualSize/random-message-loader/tree/message-ids) branch.

Documentation already available in the [documentation](/RodentOfUnusualSize/random-message-loader/tree/documentation) branch. (See commit 1d958cb7b790b8c0138d1501dd73d4e3291a5dc8.)

**Comment by RodentOfUnusualSize** [comment]
Implementation merged to main in 20c93c9c0de9d468f45e51fb07636e0e0a3bf5fd.
